src/repository/storage/postgres/orm/sequelize/note.ts

```diff
@@ -258,10 +234,10 @@ export default class NoteSequelizeStorage {
    * @param publicId - note public id
    * @returns { Promise<Note | null> } found note
    */
-  public async getNoteByPublicId(publicId: NotesSettings['publicId']): Promise<Note | null> {
+  public async getNoteByPublicId(publicId: NotePublicId): Promise<Note | null> {
     const note = await this.model.findOne({
       where: {
-        '$notes_settings.public_id$': publicId,
+        publicId,
       },
       include: {
```

Comment:
It is not required for now

Reply:
yes, but I'll remove this method later via separate PR to not to blow up the PR